### ISSUE TITLE
Update 'mature' field in API responses with 'sensitivity'

### ISSUE THREAD

**Issue by dhruvkb** [WordPress/openverse#2549]
## Problem

The implementation plan of [sensitive content filtering](https://docs.openverse.org/projects/proposals/trust_and_safety/detecting_sensitive_textual_content/20230330-implementation_plan_filtering_and_designating_results_with_sensitive_textual_content.html#derive-the-list-of-sensitive-document-ids-based-on-the-filtered-index-overview-step-4) on the backend specifies that the 'mature' field in the response will be superseded by the 'sensitivity' field which will be an array field that describes the reasons a particular piece of media is flagged as sensitive.

## Description

This field can contain values like

- user_reported_sensitive
- provider_supplied_sensitive
- sensitive_text

Based on the values in this field, the content safety wall #2462 will show relevant copy to the users.

The `Media` interface in `frontend/src/types/media.ts` contains a `mature` field (`boolean`) that will be deprecated and lacks the new `sensitivity` field (`string[]`) that must be added. Additionally all references to this `mature` field must be updated to reference the new `sensitivity` field.

Additionally the `makeFakeSensitive` utility (for testing) in `frontend/src/utils/content-safety.ts` must be updated to set the `sensitivity` field to fake values instead of the `mature` field as it does currently.

**Comment by obulat** [comment]
@dhruvkb, could you confirm what needs to be done here, if anything?

The API already returns `unstable__sensitivity` field that can have the values listed in this issue (see a sample response https://api.openverse.engineering/v1/images/).

The frontend uses the value of `unstable__sensitivity`(the value of the [`SENSITIVITY_RESPONSE_PARAM` constant](https://github.com/WordPress/openverse/blob/d55e8c9aa2784cac54f707dd4c7414c75f7857d0/frontend/src/constants/content-safety.ts#L33)) to populate the `sensitivity` parameter of the media item unless the `fake_sensitive` feature flag is turned on. 
`makeFakeSensitive` was updated to set the `sensitivity` field instead of the `mature` field

https://github.com/WordPress/openverse/blob/d55e8c9aa2784cac54f707dd4c7414c75f7857d0/frontend/src/utils/decode-media-data.ts#L106-L111

**Comment by dhruvkb** [comment]
You're correct @obulat, the work described in this issue has already been done. We can close this issue as completed.
